cluster/ci/config/prow/plugins.yaml

```diff
@@ -199,3 +199,9 @@ plugins:
   - trigger
   - wip
   - yuks
+
+  kubernetes-incubator/descheduler:
+  - assign
```

Comment:
So I am pretty sure the k8s robots run on this repo, so we should limit our plugin list to `trigger` only or our bots will race & fight with the k8s bots on doing things like assignment, labels, etc